README.md

```diff
@@ -28,6 +28,19 @@ npm run serve
 
 You should now have the application running and accessible at <http://localhost:8080> (note: will run on a different port if 8080 is occupied, most common alternative port is 8081).
 
+## Building as web component
+To build the project as a web component, run the command `npm run build-component`. This will create a file in the `dist` folder named `license-chooser.min.js`. It can be used to load the web-component in any JS project. There is also a sample `demo.html` created.
+To be able to use the file it should either be rendered statically from the integrater's web-app or be published on a CDN. Following code can be used to integrate this in other apps:
+
+```
+<script src="https://unpkg.com/vue"></script>
+<script src="https://unpkg.com/@creativecommons/chooser/dist/license-chooser.js"></script>
+
+<license-chooser></license-chooser>
+```
```

Comment:
The `demo.html` displays unstyled output:

![image](https://user-images.githubusercontent.com/17307/150772273-2523e221-3ecd-455c-a16f-9f75163df853.png)

We may need to update the demo and instructions in the README to import the Vocabulary styles.

Reply:
@brylie I am able to use demo.html with the above steps

```
<script src="https://unpkg.com/vue"></script>
<script src="http://127.0.0.1:8081/license-chooser.js"></script>

<license-chooser></license-chooser>
```

I saw that some test cases were failing. currently, pushed changes for that .

Reply:
I reset my repo and things work, aside from a couple of visual bugs.